package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "demoblaze.com",
+  "version": "1.0.0",
+  "description": "Cypress automation project for demoblaze.com",
+  "main": "index.js",
+  "scripts": {
+    "open": "npx cypress open",
+    "tsc": "npx tsc --project tsconfig.json",
+    "lint": "eslint . --ext .ts",
+    "check":"npm run tsc && npm run lint && npm run open "
```

Comment:
I mean two npm commands to run tests. For example:
"open": "npm run tsc && npm run lint && npx cypress open "
"run": "npm run tsc && npm run lint && npx cypress run "